x-pack/plugins/infra/public/components/asset_details/tabs/profiling/description_callout.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiButton, EuiCallOut, EuiFlexGroup, EuiLink, EuiSpacer } from '@elastic/eui';
+import useLocalStorage from 'react-use/lib/useLocalStorage';
+
+const LOCAL_STORAGE_KEY = 'infra-profiling-description-callout-dismissed';
+
+export function DescriptionCallout() {
+  const [isDismissed, setIsDismissed] = useLocalStorage(LOCAL_STORAGE_KEY, false);
+
+  const onDismissClick = useCallback(() => setIsDismissed(true), [setIsDismissed]);
+
+  if (isDismissed) {
+    return null;
+  }
+
+  return (
+    <EuiCallOut
+      title={i18n.translate('xpack.infra.profiling.descriptionCallout.title', {
+        defaultMessage: 'Displaying Resource Consumption for this Host',
+      })}
+      iconType="iInCircle"
+    >
+      <p>
+        {i18n.translate('xpack.infra.profiling.descriptionCallout.body', {
+          defaultMessage:
+            'Universal Profiling allows you to see what is consuming resource on your host, right down to the application code, kernel and third-party-libraries, to help you identify resource bottlenecks and optimise usage.',
+        })}{' '}
```

Comment:
```suggestion
        })}
```

Reply:
It is needed unfortunately to have a space between text and the link. But I've changed it to use `<FormattedMessage>`, it looks better now I think.